### PR TITLE
Feature/51 support sampler and estimator

### PR DESCRIPTION
**Pull request by BonnetGa**

### CODE REVIEW

File: qunicorn_core/core/jobmanager/jobmanager_service.py [hunk 1]
```diff
@@ -21,40 +21,36 @@
...
 
-    device = job_core_dto.executed_on
-    if device.provider.name == ProviderName.IBM:
-        pilot: QiskitPilot = qiskitpilot("QP")
+    if job_core_dto.executed_on.provider.name == ProviderName.IBM:
+        pilot: QiskitPilot = QiskitPilot("QP")
         pilot.execute(job_core_dto)
     else:
-        print("No valid target specified")
+        print("WARNING: No valid target specified")
```
[the remark this answers is not in the document]
Comment (BonnetGa) [reply qunicorn_core/core/jobmanager/jobmanager_service.py:41]: Würde dazu eine neue Story machen. Wie man den Logger richtig konfiguriert nutzt und so :)

**Comment by ArneBartenbach** [comment]
JobRequestDto - circuits List expression not allowed in type annotation -> use List[T]

**Comment by ArneBartenbach** [comment]
getting AccountNotFoundError (potentially caused by IBM and not the project) when creating and running jobs

**Comment by ArneBartenbach** [comment]
after testing again (without changes) the problem resolved

**Review by ArneBartenbach — APPROVED** [review]

**Pull request merged**